src/exchange/information/information.controller.js

```diff
@@ -80,10 +80,16 @@ angular
         }
 
         shouldDisplaySSLRenew () {
+            const now = moment();
+            const sslExpirationDate = moment(this.exchange.sslExpirationDate);
+            const aMonthBeforeSSLExpirationDate = sslExpirationDate.subtract(30, "days");
```

Comment:
Is there any reason to not using `.subtract(1, "months")` for instance?

Reply:
No and it actually makes much more sense, I'm gonna fix it